File: ios/Crew/Api/HttpStatus.swift
// The one Swift file that names HTTP status codes (docs/api.md conventions). Not spec numbers, so not in
// spec-constants.json; exempt from the doctrine literal scan by name (shared/scripts/doctrine-lint.mjs).
// WRITTEN — UNVERIFIED (needs Mac).

import Foundation

enum HttpStatus {
    static let ok = 200
    static let created = 201
    static let successRange = 200..<300
    static let badRequest = 400
    static let unauthorized = 401
    static let forbidden = 403
    static let notFound = 404
    static let conflict = 409
    static let tooManyRequests = 429
    static let serviceUnavailable = 503
}

File: web/src/lib/http-status.ts
// The one file that names HTTP status codes (docs/api.md conventions). Not spec numbers, so not in
// spec-constants.json; exempt from the magic-number rule by name in eslint.config.mjs.
export const HttpStatus = {
  ok: 200,
  created: 201,
  accepted: 202,
  noContent: 204,
  badRequest: 400,
  unauthorized: 401,
  forbidden: 403,
  notFound: 404,
  conflict: 409,
  payloadTooLarge: 413,
  tooManyRequests: 429,
  internal: 500,
} as const;
